content/docs/studio/user-guide/projects-and-experiments/what-is-a-project.md

```diff
@@ -24,26 +24,31 @@ To **display your project's content** in Iterative Studio,
 create `dvc.yaml`. When running model training and evaluation, save metrics and
 plots in the files defined in `dvc.yaml`.
 
+You can also use the `dvc exp push` command to **push experiments without
+creating separate Git commits** for them. To notify Iterative Studio when you
+push experiments using `dvc exp push`,
+[configure the `DVC_STUDIO_TOKEN` environment variable](/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots#set-up-an-access-token).
```

Comment:
```suggestion
[configure the Studio token](/doc/user-guide/project-structure/configuration#studio).
```

I thought it might be better to update the link since the updated link lists both `dvc config studio.token` and `DVC_STUDIO_TOKEN` as options, but I not a strong opinion, so up to you to take or leave this one @tapadipti.

Reply:
I think that it's ok to list one of the options to keep it simpler to understand. A more advanced user is more likely to see the docs page with the options anyway. So I'm leaving out this suggestion.